pkg/core/object/replicate.go

```diff
@@ -9,41 +9,45 @@ import (
 )
 
 const (
-	currentVersion = 7 // it is also a number of fields
-)
-
-const (
-	networkMagicKey = "network"
+	// required fields.
 	cidKey          = "cid"
 	oidKey          = "oid"
 	sizeKey         = "size"
-	deletedKey      = "deleted"
-	lockedKey       = "locked"
 	validUntilKey   = "validUntil"
+	networkMagicKey = "network"
```

Comment:
motivated movement?

Reply:
the same as in https://github.com/nspcc-dev/neofs-node/pull/3063#discussion_r1897966594

ok-ok, waste of time, more code to review, etc. but i feel better this way